examples/sycl/run-llama2.sh

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
Add to all `.sh` scripts

```suggestion
#!/bin/bash

```